tooling/bundler/src/bundle/windows/msi/wix.rs

```diff
@@ -637,6 +637,14 @@ pub fn build_wix_app_installer(
     data.insert("feature_group_refs", to_json(&wix.feature_group_refs));
     data.insert("feature_refs", to_json(&wix.feature_refs));
     data.insert("merge_refs", to_json(&wix.merge_refs));
+    if wix.per_machine {
+      data.insert("per_machine", to_json(true));
+      data.insert("install_scope", to_json("perMachine"));
+      data.insert("registry_root", to_json("HKLM"));
+    } else {
+      data.insert("install_scope", to_json("perUser"));
+      data.insert("registry_root", to_json("HKCU"));
+    }
```

Comment:
The else branch (or the default for that matter) needs to be moved out of the `if let Some(wix)` check. Otherwise the bundler will crash if there is no wix property in tauri.conf.json

Reply:
thanks, I fixed it locally but hasn't pushed it yet. there is more issues to solve